rwengine/src/data/ModelData.hpp

```diff
@@ -249,6 +249,44 @@ class SimpleModelInfo : public BaseModelInfo {
         return related_;
     }
 
+    static bool isDoorModel(std::string m) {
+        if (m == "oddjgaragdoor") return true;
```

Comment:
As a way to separate the data here, the list of names can be std::(unordered_)set.

Reply:
Will do